backend/tracim_backend/applications/agenda/lib.py

```diff
@@ -696,15 +696,20 @@ def sync_user_agenda(self, user: User, context: TracimContext, create_event: boo
                 logger.exception(self, exc)
 
     def sync_workspace_symlinks(
-        self, role: UserConfigInWorkspace, context: TracimContext, role_deletion=False
+        self,
+        user_workspace_config: UserConfigInWorkspace,
```

Comment:
check if we can rename the class UserConfigInWorkspace to UserWorkspaceConfig

Reply:
done